src/components/course/CourseSkills.jsx

```diff
@@ -27,12 +30,15 @@ export default function CourseSkills() {
             key={skill.name}
             placement="top"
             overlay={(
-              <Popover id="popover-positioned-top" style={{ maxWidth: 380 }}>
+              <Popover id="popover-positioned-top" style={{ maxWidth: 460 }}>
                 <Popover.Title as="h5">{skill.name}</Popover.Title>
                 <Popover.Content
                   className={classNames({ 'text-muted': !skill.description, 'font-italic': !skill.description })}
                 >
-                  { skill.description ? skill.description : SKILL_DESCRIPTION_PLACEHOLDER }
+                  {
+                    skill.description ? shortenString(skill.description, SKILL_DESCRIPTION_CUTOFF_LIMIT, ELLIPSIS_STR)
+                      : SKILL_DESCRIPTION_PLACEHOLDER
```

Comment:
So we will not show the whole description? Is there a way that a learner can still see the complete description?

Reply:
As of now, there is no way. But this cutoff is only going to happen for descriptions that have character count > 950 which fall in 95-99th percentile according to Nathan's finding.